phpstan.neon.dist

```diff
@@ -36,6 +37,7 @@ parameters:
   stubFiles:
     - tests/phpstan/stubs/wordpress-override.php
   bootstrapFiles:
+    - tests/phpunit/unit/bootstrap.php
```

Comment:
This should not be in `bootstrapFiles` but if anything just in `scanFiles`

Reply:
Removed.